libs/shared/notifier/src/lib/notifier.service.ts

```diff
@@ -16,8 +16,8 @@ export class NotifierService {
   private readonly config: NotifierSnsConfig;
 
   constructor(
-    configService: ConfigService,
```

Comment:
Is `@Inject(...)` going to be necessary for everything consumed by Auth? We haven't had to do that over on the Subplat side so far as long as something is marked as `@Injectable`, is that still the case?

Reply:
Yes, I got an error without it, which is a little strange, because as you point out it doesn't seem to have been necessary in other projects. I wonder if there's a better solution. I can try removing it, and see if I can get it working without adding injects.

Reply:
This is apparently because `esbuild` does not properly support emitting metadata, so the decorators are only spotted and the information available in some parts:
https://github.com/evanw/esbuild/issues/257

Switching to swc to run this should remedy it. Sigh.